ci/bitrise-setup-readme.md

```diff
@@ -0,0 +1,100 @@
+# Nodes iOS Bitrise - Setup Guide 🛠
+
+## Xcode
+
+Make sure your project's schemes are ***shared (including app extensions)*** , otherwise you will run into a lot of warnings and it might not build all of your targets.
+
+
+## Bitrise
+
+1) Create a *New App* from the Bitrise dashboard
+- Make sure you choose **Nodes** account when creating the *New App* and make it **Private**
+
+<p align="center">
+  <img src="../images/ci/01-new-app-bitrise.jpg?raw=true" alt="New App"/>
+</p> 
+
+2) Connect the repository to Bitrise
+
+<p align="center">
+  <img src="../images/ci/02-new-app-bitrise.jpg?raw=true" alt="New App"/>
+</p> 
+
+3) When setting up the access make sure to auto-add SSH key
```

Comment:
I think this is wrong, as we need to add the custom SSH key in order to be able to access the repo with Match certs/profiles.